app/views/redlands/related_identifier/_edit_array_hash_form.html.erb

```diff
@@ -8,17 +8,8 @@
     <%= text_field_tag "#{template}[related_identifier][][related_identifier]", hash.dig("related_identifier"),
                        class: "#{template}_related_identifier form-control multi-text-field multi_value related_identifier",
                        name: "#{template}[related_identifier][][related_identifier]",
-                        placeholder: t("simple_form.placeholders.defaults.related_identifier")
+                       placeholder: t("simple_form.placeholders.defaults.related_identifier")
```

Comment:
This is just a suggestion, but I like creating helpers to define these inputs into Helpers and keep the views clean and readable:

`def related_identifier_input_for(template, value)`
`    text_field_tag "#{template}[related_identifier][][related_identifier]", hash.dig("related_identifier"), ...`
`end`